enhancements/ingress/host-network-configurable-ports.md

```diff
@@ -0,0 +1,298 @@
+---
+title: ingress-host-network-configurable-ports
+
```

Comment:
that's extremely picky, but what's up with all those newlines here? this seems to also prevent the markdown renderer to show this as a table